microindex/finder.go

```diff
@@ -23,22 +23,20 @@ type Finder struct {
 }
 
 // NewFinder returns an object that is used to lookup keys in a microindex.
-func NewFinder(zctx *resolver.Context, uri iosrc.URI) *Finder {
-	return &Finder{
-		zctx: zctx,
-		uri:  uri,
+// It opens the file and reads the trailer, returning errors if the file is
+// corrupt, doesn't exist, or if its microindex trailer is invalid.  If the
```

Comment:
```suggestion
// It opens the file and reads the trailer, returning errors if the file is
// corrupt, doesn't exist, or has an invalid microindex.  If the
```
For parallel structure.